pyaedt/modules/Material.py

```diff
@@ -297,6 +297,16 @@ def type(self, type):
         elif self._type == "nonlinear":
             self._property_value = [self._property_value[0]]
 
+    @property
+    def evaluated_value(self):
+        """Evaluated value
+
+        Returns
+        -------
+        float
```

Comment:
```suggestion
        float
            Evaluated value.
```
I don't think properties need to have a Returns section.

Reply:
yes. Of course. Thanks